kiwi/builder/container.py

```diff
@@ -57,6 +60,18 @@ def __init__(self, xml_state, target_dir, root_dir):
         self.target_dir = target_dir
         self.container_config = xml_state.get_container_config()
         self.requested_container_type = xml_state.get_build_type_name()
+        self.base_image = None
+        image_uri = xml_state.build_type.get_derived_from()
+        if image_uri:
+            uri = Uri(image_uri, 'images')
```

Comment:
I was looking which repo_type 'images' classifies and found that we actually don't have a handling for it. What do you think about the idea to allow repo_type=None in the Uri constructor ? I have prepared a pull request to address this